src/include/miopen/general_tensor_reorder_sol.hpp

```diff
@@ -26,10 +26,10 @@
 #ifndef GUARD_GENERAL_MIOPEN_TENSOR_REORDER_SOL_HPP
 #define GUARD_GENERAL_MIOPEN_TENSOR_REORDER_SOL_HPP
 
-#include <miopen/miopen.h>
 #include <miopen/kernel_info.hpp>
 #include <miopen/op_kernel_args.hpp>
 #include <miopen/execution_context.hpp>
+#include <cstdlib>
```

Comment:
I'm not pretty sure. It seems like that we need a ```<cstdint>``` here.

Reply:
Yes, I think `<cstdint>` is the proper header file here, changed it in the latest commit.